docs/docs-beta/sidebars.ts

```diff
@@ -278,9 +278,18 @@ const sidebars: SidebarsConfig = {
           ],
         },
         {
-          type: 'doc',
+          type: 'category',
```

Comment:
something funny is happening in the sidebar for me. With the guide open i see this 
<img width="642" alt="Screenshot 2024-08-30 at 11 34 24 AM" src="https://github.com/user-attachments/assets/83e3acff-d775-4190-94dd-57b89e103d72">

with "Setting up Branch Deployments" highlighted twice. Is that intentional?

Reply:
hm interesting -- no idea why that's the case. kinda interesting as it does seem to fit nicely in either of those, although I guess putting it under branch deployments makes a bit more sense.

Reply:
yeah it's actually kind of interesting to have in in both spots in the sidebar from a finding-the-doc-youre-looking-for perspective, but strange to have both highlighted